ARM-wvd-templates/AddVirtualMachinesToHostPool/README.md

```diff
@@ -0,0 +1,15 @@
+Click the button below to deploy:
+
+<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2FRDS-Templates%2Fmaster%2FARM-wvd-templates%2FUpdate%20existing%20WVD%20host%20pool%2FAddVirtualMachinesTemplate.json" target="_blank">

```

Comment:
This needs to be updated with the new parent folder name

Reply:
udpated